src/main/java/com/octopus/bamboo/plugins/task/createrelease/CreateReleaseTaskConfigurator.java

```diff
@@ -50,6 +50,8 @@ public CreateReleaseTaskConfigurator(@NotNull final TextProvider textProvider,
         config.put(OctoConstants.SPACE_NAME, params.getString(OctoConstants.SPACE_NAME));
         config.put(OctoConstants.PROJECT_NAME, params.getString(OctoConstants.PROJECT_NAME));
         config.put(OctoConstants.CHANNEL_NAME, params.getString(OctoConstants.CHANNEL_NAME));
+        config.put(OctoConstants.GIT_REF, params.getString(OctoConstants.GIT_REF));
```

Comment:
not sure if it would have been possible - but if the OctoConstants had been an Enum, this could have been a loop :/